src/resolve-dependency.ts

```diff
@@ -176,6 +183,12 @@ function resolvePackage (name: string, parent: string, job: Job, cjsResolve: boo
     }
   }
 
+  if (typeof resolveRequest === 'function') {
+    try {
+      return resolveRequest(name, parent);
```

Comment:
Can't this function just use `require.resolve(name, { paths: [parent] });` to get the entry point for each `name`? That would make it not be PnP specific

Reply:
that is a good idea, let me check that to see if it works @merceyz.